packages/block-editor/src/components/plain-text/style.native.scss

```diff
@@ -1,5 +1,5 @@
 
-.block-editor-plain-text {
+.block-editor-plain-text textarea {
```

Comment:
This is necessary because `TextareaControl` has a couple of wrappers around the `textarea` element.